sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/plans/PlanTest.scala

```diff
@@ -26,6 +26,9 @@ import org.apache.spark.sql.catalyst.util._
  * Provides helper methods for comparing plans.
  */
 abstract class PlanTest extends SparkFunSuite with PredicateHelper {
+
+  System.setProperty("spark.testing", "true")
```

Comment:
This is already set by the test framework. MIght be cleaner not to hard code it in just a few places in the tests.